recipes/journey_1/module_3/probe.yaml

```diff
@@ -223,15 +278,25 @@ tasks:
             - type: "text"
               content: "Here comes the next one, {username}! Let's explore the importance of context in prompt engineering."
             - type: "text"
-              content: "Why is context important in prompt engineering?\n\nA. It increases the length of the prompt.\nB. It helps the model generate relevant content by establishing the setting and background.\nC. It makes the prompt more complex.\nD. It limits the model's output."
-        edges:
-          - text: "a"
+              content: |
```

Comment:
Example of Correct YAML style. There exist the end node and appropriately no 2nd message before the question

![image](https://github.com/user-attachments/assets/2fcc7734-b803-4b74-8bf7-430d7cadcaef)